draft-duke-quic-load-balancers.md

```diff
@@ -310,18 +285,19 @@ depicted in the figure below.
 ~~~~~
 {: #plaintext-cid-format title="Plaintext CID Format"}
 
-### Load Balancer Actions
+### Configuration Agent Actions
+
+The configuration agent selects a number of bytes of the server connection ID
+(SCID) to encode individual server IDs, called the "routing bytes". The number
+of bytes MUST have enough entropy to have a different code point for each
+server.
 
-The load balancer selects a number of bytes of the server connection ID
-(SCID) that it will use to route to a given server, called the "routing bytes".
-The number of bytes MUST have enough entropy to have a different code point for
-each server.
+It also assigns a server ID to each server.
 
-The load balancer shares this value with servers, as explained in
-{{protocol-description}}, along with the value that represents that server.
+### Load Balancer Actions
 
 On each incoming packet, the load balancer extracts consecutive octets,
-beginning with the second byte. These bytes represent the server ID.
+beginning with the second. These bytes represent the server ID.
```

Comment:
I think it sounds better with something after `second`. Either `byte` or `one` or something.